website/src/releases/2023-03/button-sizes.mdx

```diff
@@ -0,0 +1,34 @@
+---
+date: TBA
+title: Unifying all UI Kit button sizes
+description: Introducing consistent sizes for all the UI Kit buttons.
+type: feature
+topics:
+  - Merchant Center
+  - UI Components
```

Comment:
We've updated the [list of topics](https://commercetools.atlassian.net/wiki/spaces/DOCS/pages/105818548/Writing+release+notes) for Custom Application release notes.

```suggestion
  - Merchant Center
  - Development
```

Reply:
Looks good to me 👍